### DISCUSSION TITLE
Sql connection pooling

### DISCUSSION THREAD

**Discussion by dcbrown16** [Azure/azure-functions-durable-extension#2210] [Q&A]
Can we use SQL connection pools in C# Durable Functions?
I see the guidance here that it is possible to use connection pools in Functions in general, but we are wondering if there are any specific points to consider for Durable Functions.
https://docs.microsoft.com/en-us/azure/azure-functions/manage-connections?tabs=csharp

(This is for an advisory case, my team doesn't have experience with coding.)

**Comment by cgillum** [comment]
Yes, absolutely. In fact, SQL connection polling is the default behavior in .NET, regardless of whether you’re running in Functions or some other environment. Durable Functions is not special in this regard. Just make sure that you create SQL connections in activity or entity functions, and not inside orchestrator functions.
